lib/rubocop/cop/style/first_parameter_indentation.rb

```diff
@@ -39,6 +39,8 @@ def on_send(node)
         private
 
         def message(arg_node)
+          return 'Incorrect indentation of the first parameter' if arg_node.nil?
```

Comment:
The message should end with a `.` (period), and it would be good if you tested `cop.messages` in the spec to show the different messages.